src/components/TagSet/component.tsx

```diff
@@ -0,0 +1,17 @@
+import styles from "./styles.module.scss";
+import { Tag } from "../Tag";
+import React, { FunctionComponent } from "react";
+
+const TagSet: FunctionComponent<ITagSetProps> = ({ tags }) => (
+  <section className={styles.items}>
+    {tags.map((tag, index) =>
+      <Tag key={index} className={styles.item} name={tag}/>
```

Comment:
hace poquito averigue que no es recomendable usar el index como key porque es mas ineficiente si el array va cambiando

Reply:
tenes mucha razon, en este caso habría que usar el tag directamente

https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318